### PR TITLE
Add taiga solution chatflow

### PR DESCRIPTION
**Pull request by waleedhammam**
for issue: https://github.com/threefoldtech/js-sdk/issues/427

### CODE REVIEW

File: jumpscale/packages/tfgrid_solutions/chats/taiga_deploy.py [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+    def select_pool(self):
+        cu, su = deployer.calculate_capacity_units(**self.query)
+        self.pool_id = deployer.select_pool(self, cu=cu, su=su, **self.query)
+
+    @chatflow_step(title="Network")
+    def taiga_network(self):
+        self.network_view = deployer.select_network(self)
+
+    @chatflow_step(title="Access keys")
+    def public_key_get(self):
```
Comment (dmahmouali) [inline jumpscale/packages/tfgrid_solutions/chats/taiga_deploy.py:65]: should remove this step, no?

File: jumpscale/packages/tfgrid_solutions/chats/taiga_deploy.py [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+        self.public_key = self.upload_file(
```
Comment (dmahmouali) [inline jumpscale/packages/tfgrid_solutions/chats/taiga_deploy.py:66]: unused variable

**Comment by waleedhammam** [comment]
it's still a draft, has some local changes

**Comment by waleedhammam** [comment]
reopend to refactor to be aliened with other apps

File: jumpscale/packages/admin/frontend/data.js [hunk 3]
```diff
@@ -156,6 +156,14 @@ const SOLUTIONS = {
     url: "/solutions/cryptpad_deploy",
     description: "CryptPad is the Zero Knowledge realtime collaborative editor."
   },
+  taiga: {
+    topic: "taiga_deploy",
+    type: "taiga",
+    name: "Taiga",
+    image: "./assets/taiga.png",
+    url: "/solutions/taiga_deploy",
+    description: "Build your all in one taiga solution"
```
Comment (AhmedSa-mir) [inline jumpscale/packages/admin/frontend/data.js:165]: all what? :D

Comment (waleedhammam) [reply jumpscale/packages/admin/frontend/data.js:165]: all in one :D, means taiga integrated with threefold stuff

File: jumpscale/packages/marketplace/chats/taiga.py [hunk 4]
```diff
@@ -0,0 +1,326 @@
...
+            )
+            if result:
+                self.md_show_update("Deploying Network on Nodes....")
+                for wid in result["ids"]:
+                    success = deployer.wait_workload(wid)
+                    if not success:
+                        raise StopChatFlow(f"Failed to add node {self.selected_node.node_id} to network {wid}")
+                self.network_view = self.network_view.copy()
+            self.ip_address = self.network_view.get_free_ip(self.selected_node)
+        self.container_ip()
```
Comment (AhmedSa-mir) [inline jumpscale/packages/marketplace/chats/taiga.py:155]: this method calls add_network_node again? it's already added above, no?

File: jumpscale/packages/marketplace/frontend/components/solutions/Solutions.vue [hunk 5]
```diff
@@ -92,6 +92,12 @@ module.exports = {
           description:
             "Gitea is a painless self-hosted Git service. It is similar to GitHub, Bitbucket, and GitLab.",
         },
+        {
+          type: "taiga",
+          name: "Taiga",
+          image: "./assets/taiga.png",
+          description: "Build your all in one taiga solution.",
```
Comment (AhmedSa-mir) [inline jumpscale/packages/marketplace/frontend/components/solutions/Solutions.vue:99]: Here also :D

File: jumpscale/packages/admin/frontend/data.js (hunk 3 above)
Comment (AhmedSa-mir) [reply jumpscale/packages/admin/frontend/data.js:165]: ah ok. should be all-in-one?

**Pull request merged**